ansible_runner/__main__.py

```diff
@@ -49,6 +50,106 @@
 logger = logging.getLogger('ansible-runner')
 
 
+class Bunch(object):
```

Comment:
This feels a little dirty to me... I'd almost rather just see direct dict mappings or some other way (namedtuples?) to map to properties.

I do like the usage of role_manager as a `contextmanager` though. We should keep that.